packages/react/src/AnchorNav/AnchorNav.module.css

```diff
@@ -0,0 +1,340 @@
+/*
+ * AnchorNav
+ * 
+ * 1. Globals
+ * 2. AnchorNav root and inner bounding containers
+ * 3. AnchorNav links
+ * 4. AnchorNav link labels
+ * 5. AnchorNav menu toggle button (narrow)
+ * 6. AnchorNav action
+ * 7. Custom animations
+ */
+
+/*
+ * 1. Globals
+ * TODO: Move into global reset
+ */
+:global(html) {
+  scroll-behavior: smooth;
+}
+
+@media screen and (prefers-reduced-motion: reduce) {
+  :global(html) {
+    scroll-behavior: auto;
+  }
+}
+
+/*
+ * 2. AnchorNav root and inner bounding containers
+ */
+.AnchorNav {
+  display: flex;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: var(--base-size-12) var(--base-size-16);
+  z-index: 91; /* must be higher than subdomain nav bar */
+}
+
+.AnchorNav--with-default-background-color {
+  background-color: var(--brand-color-canvas-default);
+}
+
+.AnchorNav--stuck {
+  box-shadow: 0 3px 6px rgba(140, 149, 159, 0.15), 0 1px 1px rgba(0, 0, 0, 0.1); /* TODO: replace with a global token */
+}
+
+.AnchorNav--expanded {
+  transform-style: preserve-3d;
+  z-index: 91;
+}
+
+.AnchorNav-link-container {
+  display: flex;
+  width: 100%;
+  position: relative;
+}
+
+.AnchorNav-inner-container {
+  max-width: 1280px;
+  width: 100%;
+  position: relative;
+  z-index: 91; /* must be higher than subdomain nav bar */
+}
+
+.AnchorNav--with-default-background-color .AnchorNav-inner-container {
+  background-color: var(--brand-color-canvas-default);
+}
+
+/* Up to large breakpoint */
+@media screen and (max-width: 63.25rem) {
+  .AnchorNav {
```

Comment:
On mobile, do you think we should add something like `touch-action: none` to prevent users from accidentally scrolling the page when the menu is toggled?

Video shows how I can still scroll behind the menu when it is visible:

https://user-images.githubusercontent.com/1907033/206274702-0403b832-2564-4598-b349-47b78da151a3.mov

Reply:
Thanks @xodene. This is actually intentional to match the behaviour on github.com/features and resources.github.com. @danielguillan, what do you think? Keep as is or lock it? As another idea, perhaps we enable it an optional prop.

Reply:
It makes sense to lock the page scrolling under the backdrop, but I'm not sure what the behavior should be when the component isn't stuck. We need to do a quick prototype and test before we make a decision.